src/lib/utils/calibrated_salinity.h

```diff
@@ -0,0 +1,68 @@
+#include <math.h>
+
+double calculate_calibrated_salinity(const double measured_conductivity, const double temperature,
```

Comment:
I think we need unit test for this.

Reply:
Added a Boost unit test to the `src/tests/utils` directory.